src/components/form/list.vue

```diff
@@ -106,7 +106,7 @@ export default {
     },
     "emptyTable": "There are no Forms to show.",
     "alert": {
-      "create": "Your new Form “{name}” has been created as a Draft. Upload files and test your Form below, and when you feel it’s ready, you can publish the Form for use."
+      "create": "“{name}” has been created as a Form Draft. Upload files to test your Form."
```

Comment:
@ktuite says that it doesn't make sense to upload files _to_ test your Form: that's not the purpose of uploading the files. Suggestion: either replace "to" with "and" or remove the second sentence altogether.

Reply:
We're going to remove the second sentence.